modules/manage/pages/security/fips-compliance.adoc

```diff
@@ -0,0 +1,42 @@
+= Configure Redpanda for FIPS 
+:description: Configure Redpanda to operate in FIPS compliance mode.
+:page-aliases: security:fips-compliance.adoc
+:page-categories: Management, Security
+
+Redpanda platform (Redpanda Enterprise) provides FIPS-compliant cipher enforcement for brokers using OpenSSL 3.0.9, which is https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/4282[validated^] for https://csrc.nist.gov/pubs/fips/140-2/upd2/final[140-2^] and is undergoing validation by NIST for https://csrc.nist.gov/pubs/fips/140-3/final[140-3^]. Both Redpanda and `rpk` leverage validated Open SSL libraries for all security-related cryptography operations.
+
+== Prerequisites
+
+Before configuring brokers to run in FIPS compliance mode (FIPS mode), check to make sure the `redpanda-rpk-fips` and `redpanda-fips` packages are xref:deploy:deployment-option/self-hosted/manual/production/dev-deployment.adoc#install-fips[installed]. These packages are required by both the `redpanda` and `redpanda-tuners` install packages.
+
+== Configure FIPS
+
+When you configure a broker to run in FIPS mode:
+
+* FIPS compliance is enforced _immediately_ across all services upon startup.
+
+* Redpanda and its dependencies only use FIPS-validated cryptographic modules for all cryptographic algorithms used in a security context.
+
+Redpanda logs an error and exits immediately if:
+
+* The underlying operating system and crypto module are not running in FIPS mode.
+
+* The underlying cryptography module enters into an error state.
+
+* It cannot detect a FIPS-validated library.
+
+Additionally, a Redpanda broker will not join a cluster of brokers running in a different FIPS mode of operation. Instead, it logs an error and exits immediately.
+
+To place a broker in FIPS compliance mode, enable `fips_mode` in the Redpanda broker configuration file (typically located in `/etc/redpanda/redpanda.yaml`). All fields are within the `redpanda` object:
```

Comment:
`fips_mode` supports three values:

* `disabled` - self-explanatory
* `enabled`
* `permissive`

The difference between `enabled` and `permissive` is how it handles the following:

When Redpanda starts up, it looks for the file `/proc/sys/crypto/fips_enabled` and reads it and expects the contents to equal "1".

In `enabled` mode, if the file doesn't exist or doesn't contain "1", then Redpanda logs an error and exits immediately.

In `permissive` mode, Redpanda logs a WARNING but continues operations.

Reply:
To add my two cents. 

`fips_mode=permissive` is a safety value and _shouldn't_ be used in a production environment. It's for folks who may have an edge case and know what they are doing. This will allow Redpanda to start even if the underlying operating system is not configured for FIPS.

Most customers are better off only using `fips_mode=enabled` if they need FIPS so that they don't assume compliance because Redpanda started.

Reply:
We should also call out setting up the `openssl_config_file` and `openssl_module_directory` options:
https://redpandadata.atlassian.net/browse/DOC-148?focusedCommentId=27398

Look for the FIPS configuraiton section of that comment

Reply:
```
redpanda:
  # ....
  fips_mode: enabled
  openssl_config_file: /opt/redpanda/openssl/openssl.cnf
  openssl_module_directory: /opt/redpanda/lib/ossl-modules/
```